### PR TITLE
adding actionbar

### PR DESCRIPTION
**Pull request by cixzhang**
Didn't change the styling/behavior yet. This just makes it a module. Also removed the example.

### CODE REVIEW

File: src/actionbar.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+var template = `
+  <button class="reset" type="button">reset</button>
+  <button class="prev" type="button">&lt;</button>
+  <button class="next" type="button">&gt;</button>
+  <input class="frame" disabled></input>
+`;
```
Comment (ptpham) [inline src/actionbar.js:7]: Awww, yis.

File: src/viewer.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ var Viewer = function (el) {
   return this;
 };
 
+Viewer.Actionbar = require('./actionbar.js');
```
Comment (ptpham) [inline src/viewer.js:19]: Maybe like ... ActionBar one day?

Comment (cixzhang) [inline src/viewer.js:19]: Aww okay.

Comment (ptpham) [inline src/viewer.js:19]: Oh, in case it wasn't clear, that was a non-blocking suggestion. =P

**Pull request merged**